### ISSUE TITLE
Suspicious ranges, false warning.

### ISSUE THREAD

**Issue by ivarne** [tonyhffong/Lint.jl#11]
Not sure what we should do with this, but I figured an issue is the best way to keep track of it.

[`splice!`](http://docs.julialang.org/en/latest/stdlib/base/?highlight=splice!#Base.splice!) seems to use backwards ranges to denote the place where something should be added.

``` julia
julia> x = [1,2,7,8];
julia> y = [3,4,5,6];

julia> splice!(x, 3:2, y)
0-element Array{Int64,1}

julia> x
8-element Array{Int64,1}:
 1
 2
 3
 4
 5
 6
 7
 8
```

ref: https://groups.google.com/forum/#!topic/julia-users/OhmzrjPun3I
